doc/sphinx/changes.rst

```diff
@@ -8,6 +8,687 @@ Recent changes
+  `-native-compiler` option of the configure script is added as an on demand
+  value, which becomes the default, thus preserving the previous default
+  behavior.
+  The stdlib is still precompiled when configuring with `-native-compiler
+  yes`. It is not precompiled otherwise.
+  This an implementation of point 2 of
+  `CEP #48 <https://github.com/coq/ceps/pull/48>`_
+  (`#13352 <https://github.com/coq/coq/pull/13352>`_,
+  by Pierre Roux).
+- **Changed:**
+  The :attr:`export` locality can now be used for all Hint commands,
+  including :cmd:`Hint Cut`, :cmd:`Hint Mode`, :cmd:`Hint Transparent` / `Opaque` and
+  :cmd:`Remove Hints`
+  (`#13388 <https://github.com/coq/coq/pull/13388>`_,
+  by Pierre-Marie Pédrot).
+- **Removed:**
+  In the :cmd:`Extraction Language` command, remove `Ocaml` as a valid value.
+  Use `OCaml` instead.  This was deprecated in Coq 8.8, `#6261 <https://github.com/coq/coq/pull/6261>`_
+  (`#13016 <https://github.com/coq/coq/pull/13016>`_, by Jim Fehrle).
+
+  .. _813HintWarning:
+
+- **Deprecated:**
+  The default value for hint locality is currently :attr:`local` in a section and
+  :attr:`global` otherwise, but is scheduled to change in a future release. For the
+  time being, adding hints outside of sections without specifying an explicit
+  locality is therefore triggering a deprecation warning. It is recommended to
+  use :attr:`export` whenever possible
+  (`#13384 <https://github.com/coq/coq/pull/13384>`_,
+  by Pierre-Marie Pédrot).
+- **Deprecated:**
+  :cmd:`Grab Existential Variables` and :cmd:`Existential` commands
+  (`#12516 <https://github.com/coq/coq/pull/12516>`_,
+  by Maxime Dénès).
+- **Added:**
+  Added support for automatic insertion of coercions in :cmd:`Search`
+  patterns. Additionally, head patterns are now automatically
+  interpreted as types
+  (`#13255 <https://github.com/coq/coq/pull/13255>`_,
+  fixes `#13244 <https://github.com/coq/coq/issues/13244>`_,
+  by Hugo Herbelin).
+- **Added:**
+  The :cmd:`Proof using` command can now be used without loading the
+  Ltac plugin (`-noinit` mode)
+  (`#13339 <https://github.com/coq/coq/pull/13339>`_,
+  by Théo Zimmermann).
+- **Added:**
+  Clarify in the documentation that :cmd:`Add ML Path` is not exported to compiled files
+  (`#13345 <https://github.com/coq/coq/pull/13345>`_,
+  fixes `#13344 <https://github.com/coq/coq/issues/13344>`_,
+  by Hugo Herbelin).
+
+Tools
+^^^^^
+
+- **Changed:**
+  Added the ability for coq_makefile to directly set the installation folders,
+  through the :n:`COQLIBINSTALL` and :n:`COQDOCINSTALL` variables.
+  See :ref:`coqmakefilelocal`.
+  (`#12389 <https://github.com/coq/coq/pull/12389>`_, by Martin Bodin, review of Enrico Tassi).
+- **Changed:**
+  ``dev/tools/make-changelog.sh`` now asks for a list of bugs fixed by the PR
+  (`#12410 <https://github.com/coq/coq/pull/12410>`_, fixes `#12386
+  <https://github.com/coq/coq/issues/12386>`_, by Jason Gross).
```

Comment:
It's really nice and all, but this is really an internal change of no interest to users reading the changelog, so this should not be there.
```suggestion
```

Reply:
Indeed, I thought so too.